app/presenters/notices/setup/abstraction-alerts-notifications.presenter.js

```diff
@@ -52,7 +72,69 @@ function _addressLines(contact) {
   return addressLines
 }
 
-function _email(recipient, referenceCode, eventId) {
+/**
+ * All the Water Abstraction Alerts templates require the same personalisation data:
+ *
+ * ```javascript
+ *  {
+ *    condition_text: '',
+ *    flow_or_level: 'flow',
+ *    issuer_email_address: 'defra@admin.gov.uk',
+ *    licence_ref: recipients.licenceHolder.licence_refs,
+ *    monitoring_station_name: 'Death star',
+ *    source: '',
+ *    threshold_unit: 'm3/s',
+ *    threshold_value: 100
+ *  }
+ * ```
+ *
+ * 'condition_text' can be empty, the templates is structured so when it's empty is does not affect the copy.
+ *
+ * In the case of a letter, the address is also required.
+ *
+ * @private
+ */
+
+function _commonPersonalisation(licenceMonitoringStation, monitoringStationName) {
+  return {
+    condition_text: '',
```

Comment:
The empty parts still need to be implemented. 

This will be done in other changes.